src/constants/EnemyStats.js

```diff
@@ -0,0 +1,12 @@
+export const STAT_FIELDS = [
+  {
+    label: "Name",
+    field: "enemy_name",
+    defaultValue: "Hilichurl",
+  },
+  {
+    label: "Enemy Level",
+    field: "enemy_level",
+    defaultValue: 90,
+  },
```

Comment:
add resistance

Reply:
do you want me to calculate resistance as well or juwt add the field to EnemyStats.js ?

Reply:
calc that too, and keep it a whole number but return as percentage for function